libs/datamodel/core/src/transform/ast_to_dml/validation_pipeline/validations/relations.rs

```diff
@@ -251,11 +256,15 @@ pub(super) fn cycles<'ast, 'db>(
 /// onDelete.
 pub(super) fn multiple_cascading_paths(
     relation: CompleteInlineRelationWalker<'_, '_>,
-    connector: &dyn Connector,
+    db: &ParserDatabase<'_>,
     diagnostics: &mut Diagnostics,
 ) {
+    let connector = db.active_connector();
     if !connector.has_capability(ConnectorCapability::ReferenceCycleDetection)
-        || !connector.has_capability(ConnectorCapability::ForeignKeys)
+        || db
```

Comment:
This check we're going to change on this week. Let's still keep it like this though.